### DISCUSSION TITLE
Check time ...

### DISCUSSION THREAD

**Discussion by mannbravo** [henrygd/beszel#175] [Ideas]
I am new to "uptime kuma", use for monitor network device, and so far so good.
btw ... would like to propose ideas if ...can create or change .i.e "check every 600 second" into real time (i.e minute/second)
So we can see time in progress, rather than only see "check every 600 second".

could it possible ?

![Screenshot 2024-09-20 111050 - check time](https://github.com/user-attachments/assets/be0cabf9-15fc-4cab-ab23-5dedf717153b)

**Comment by henrygd** [comment]
Maybe, but I suggest asking here instead: https://github.com/louislam/uptime-kuma
